docs/guides/applications/configuration-management/puppet/_index.md

```diff
@@ -9,6 +9,6 @@ license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
 published: 2020-07-15
 title: Puppet
 show_in_lists: true
-aliases: ['/applications/configuration-management/puppet/']
+aliases: ['/applications/configuration-management/puppet/','/application-stacks/puppet/automation/']
```

Comment:
This alias only existed on the deleted find-this-guide-on-github article. As a result, deleting that article meant the alias link would be abandoned and would now 404. I thought it could be preserved by linking it to the Puppet section

Reply:
Thanks for adding it back in!